src/modules/paymentProcessor/StreamingPaymentProcessor.sol

```diff
@@ -518,6 +518,8 @@ contract StreamingPaymentProcessor is Module, IStreamingPaymentProcessor {
             .length - 1];
 
         activeVestingWallets[client][paymentReceiver].pop();
+
+        //TODO check if event is missing?
```

Comment:
TODO

Reply:
Fixed. The event gets emitted in _afterClaimCleanup()